src/core/constraints.jl

```diff
@@ -107,3 +107,8 @@ struct ComponentActivePowerVariableLimitsConstraint <: PowerVariableLimitsConstr
 struct ComponentReactivePowerVariableLimitsConstraint <: PowerVariableLimitsConstraint end
 
 should_write_resulting_value(::Type{<:ConstraintType}) = true
+
+convert_result_to_natural_units(::Type{<:ConstraintType}) = false
+convert_result_to_natural_units(::Type{CopperPlateBalanceConstraint}) = true
```

Comment:
Why do you want conversion for these if the cost function is already in the MW-monetary unit. None of these should be converted when querying the duals.